manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java

```diff
@@ -71,6 +82,34 @@ public void send(NoticeReceiver receiver, NoticeTemplate noticeTemplate, Alert a
         }
     }
 
+    private WeWorkWebHookDto checkNeedAtNominator(NoticeReceiver receiver, Alert alert) {
+        if (StringUtils.isBlank(receiver.getPhone()) && StringUtils.isBlank(receiver.getTgUserId())) {
+            return null;
+        }
+        WeWorkWebHookDto weWorkWebHookTextDto = new WeWorkWebHookDto();
+        weWorkWebHookTextDto.setMsgtype(WeWorkWebHookDto.TEXT);
+        WeWorkWebHookDto.TextDTO textDto = new WeWorkWebHookDto.TextDTO();
+        String alertMessage = String.format("警告对象：%s\n详情：%s", alert.getTarget(), alert.getContent());
+        textDto.setContent(alertMessage);
+        if (StringUtils.isNotBlank(receiver.getPhone())) {
+            textDto.setMentioned_mobile_list(analysisArgToList(receiver.getPhone()));
+            weWorkWebHookTextDto.setText(textDto);
```

Comment:
maybe line 96 and 100 can combine into one line

Reply:
The two entries are different